x-pack/plugin/mapper-constant-keyword/build.gradle

```diff
@@ -7,7 +7,7 @@ esplugin {
   name 'constant-keyword'
   description 'Module for the constant-keyword field type, which is a specialization of keyword for the case when all documents have the same value.'
   classname 'org.elasticsearch.xpack.constantkeyword.ConstantKeywordMapperPlugin'
-  extendedPlugins = ['x-pack-core', 'lang-painless']
+  extendedPlugins = ['x-pack-core']
```

Comment:
Oh wild. So they get the extension transitively through x-pack-core -> aggregations -> lang-painless? That seems odd. Does x-pack-core extend aggregations just so it can see it or something else? I'm not entirely clear on when you'd do what with the extension mechanism.